lib/catFile.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var through = require('through2');
+var gutil = require('gulp-util');
+var exec = require('child_process').exec;
+var escape = require('any-shell-escape');
+var stripBom = require('strip-bom-stream');
+var BufferStreams = require('bufferstreams');
```

Comment:
There are now three stream libraries, through2, bufferstream and strip-bom-stream.
through2 can do what bufferstream does by just wrapping the `file.contents` in a `new Buffer()`